src/_articles/archive/zones.md

```diff
@@ -271,20 +285,20 @@ Consider this example:
 var completer = new Completer();
 var future = completer.future.then((x) => x + 1);
 var zoneFuture;
-runZoned(() {
+runZonedGuarded(() {
   zoneFuture = future.then((y) => throw 'Inside zone');
-}, onError: (error) {
-  print('Caught: $error');
-});
+}, (error) { print('Caught: $error'); });
+
 zoneFuture.catchError((e) { print('Never reached'); });
 completer.complete(499);
 {% endprettify %}
 
 Even though the future chain ends in a `catchError()`,
 the asynchronous error can't leave the error zone.
-Instead, the zoned error handler (the one specified in `onError`)
+Instead, the zoned error handler
+(the one specified in `runZonedGuarded()`)
 handles the error.
```

Comment:
**suggestion:** You don't need the instead or the parenthetical. Both imply that some of this text can be ignored. It really can't. Also, the parenthetical disrupted your flow, because otherwise "The zoned error handler handles the error" would have stood out more.

```suggestion
The zoned error handler found in `runZonedGuarded()`
responds to an error.
```